File: src/Clients/to-do-assistant/src/models/viewmodels/notification.ts
export class Notification {
  constructor(
    public id: number,
    public listId: number,
    public taskId: number,
    public userImageUri: string,
    public message: string,
    public isSeen: boolean,
    public createdDate: string,
    public formattedCreatedDate: string
  ) {}
}

File: src/ToDoAssistant/ToDoAssistant.Application/Entities/Notification.cs
using Core.Application;
using Core.Application.Entities;

namespace ToDoAssistant.Application.Entities;

public class Notification : Entity
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public int ActionUserId { get; set; }
    public int? ListId { get; set; }
    public int? TaskId { get; set; }
    public string Message { get; set; } = null!;
    public bool IsSeen { get; set; }

    public User? User { get; set; }
}
